### PR TITLE
CE, clarify platforms, alphabetize libraries

### PR DESCRIPTION
**Pull request by kwalrath**
Suggested changes to #1505.

### CODE REVIEW

File: src/_guides/libraries/index.md [hunk 1]
```diff
@@ -5,37 +5,41 @@ short-title: "Core libraries"
...
 (`dart:collection`), making calculations (`dart:math`), and encoding/decoding
-data (`dart:convert`).
+data (`dart:convert`). Additional APIs are available in
+[community contributed packages](/guides/libraries/useful-libraries).
 
-The following table shows an inventory of all Dart core libraries. Aadditional
-APIs are available in community contributed packages located [in shared
-packages](/guides/libraries/useful-libraries). Also note that some core
-libraries are only available on some [Dart platforms](/platforms).
+The following table lists all of the Dart core libraries.
```
Comment (mit-mit) [inline src/_guides/libraries/index.md]: Please add back the link to packages

File: src/_guides/libraries/index.md [hunk 2]
```diff
@@ -5,37 +5,41 @@ short-title: "Core libraries"
...
+Each library works on at least one [platform](/platforms):
+
+* **JIT:** The Dart VM in just-in-time compilation mode. Includes Flutter mobile, command-line, and server apps, unless otherwise noted.
```
Comment (mit-mit) [inline src/_guides/libraries/index.md]: Please let's discuss these over in the platforms PR; I don't like the way these are phrased. So can we please remove them for now, work on them in the platforms PR, and then add back here (possibly via an include) when they are done.

File: src/_guides/libraries/index.md (hunk 1 above)
Comment (kwalrath) [reply src/_guides/libraries/index.md]: I moved it up above.

File: src/_guides/libraries/index.md (hunk 2 above)
Comment (kwalrath) [reply src/_guides/libraries/index.md]: OK, will do. I'll edit #1505 to be a less invasive edit, perhaps with a TBD for the legend.

**Pull request closed without merging**